taskcluster/ci/push-apk/kind.yml

```diff
@@ -27,11 +28,15 @@ job-template:
     description: Publish Fenix
     worker-type: push-apk
     worker:
-        commit: true
+        commit:
+            by-build-type:
+                fennec-production: false
```

Comment:
Question: is this going to be set to true once we finalized the migration?

Reply:
Yes. I put this value, just to make sure the pipeline isn't broken before starting the migration.